Podfile

```diff
@@ -0,0 +1,16 @@
+# Uncomment this line to define a global platform for your project
+# platform :ios, '6.0'
+
+target 'OneBusAway' do
+pod 'ParseUI'
+
+end
+
+target 'Versioning' do
```

Comment:
As a point of clarification I was thinking this might be causing the Travis failure:

> $ pod install
> Analyzing dependencies
> CocoaPods 0.36.0 is available.
> To update use: `sudo gem install cocoapods`
> For more information see http://blog.cocoapods.org
> and the CHANGELOG for this version http://git.io/BaH8pQ.
> [!] Unable to find a target named `Versioning`
> The command "eval pod install" failed. Retrying, 2 of 3.